src/drivers/gitlab.js

```diff
@@ -69,6 +70,9 @@ class Gitlab {
   async commitCommentCreate(opts = {}) {
     const { commitSha, report } = opts;
 
+    if (report.length >= MAX_COMMENT_SIZE)
+      throw new Error('GitLab Comment too Large');
```

Comment:
should we warn instead & truncate the report?

incidentally we should also have a generic `warn` function that does the appropriate CI-dependent thing (e.g. on GH, https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-warning-message)

Reply:
https://github.com/iterative/cml/blob/master/src/drivers/github.js#L781

Reply:
can move to follow up.

Also better error message should suggest cause & solutions:

- split into multiple reports
- publish fewer images
- don't use `--publish-native` as it encodes images in `base64`, easily hitting the limit
- what is the max char limit (1M)